### PR TITLE
Split components

### PR DESCRIPTION
**Pull request by ooade**

### CODE REVIEW

**Comment by ooade** [comment]
I don't have vuetur installed or whatever its called on my new computer :sweat_smile: to properly align things but that's how to split things in vue

File: src/components/ReviewForm.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+	<form v-on:submit.prevent="addNewComment(newReviewComment, rating, id + 1); newReviewComment = ''; rating = '';">
```
Comment (ooade) [inline src/components/ReviewForm.vue:2]: This can be done differently :sweat_smile: 
Too lazy to do it.

Comment (Youngestdev) [reply src/components/ReviewForm.vue:2]: And I wrote a bunch of code. Someday I'll learn how to code :laughing:

**Comment by Youngestdev** [comment]
I'm waiting for the second person helping with this.. Meanwhile, I'll check this :tada:

**Pull request merged**